### PR TITLE
Logentries chart

### PR DESCRIPTION
**Pull request by heshamMassoud**
In this PR we create a logentries [daemonset.yaml](https://github.com/commercetools/k8s-charts/compare/logentries-chart?expand=1#diff-cd28b83984f42054acc56b60a85d471e) for creating helm chart for setting up logentries on k8s.

### CODE REVIEW

**Review by andrii-kovalenko-ct — APPROVED** [review]

**Review by svenmueller — COMMENTED** [review]
basically all my comments for the PR regarding the cronjobs Helm chart apply here too :)

File: logentries/templates/daemon-set.yaml [hunk 1]
```diff
@@ -0,0 +1,39 @@
+apiVersion: apps/v1
```
Comment (svenmueller) [inline logentries/templates/daemon-set.yaml]: maybe just name the file `daemonset.yaml` (without the dash)?

File: logentries/README.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+```
+$ helm upgrade --install --dry-run --debug my-new-chart  .
+```
+
+### Installation
+
+To install the chart with the release name `my-logentries` using a dedicated namespace(recommended):
+
+```
+$ helm install --name my-logentries--namespace logentries .
```
Comment (andrii-kovalenko-ct) [inline logentries/README.md]: `my-logentries--namespace` space is missing

File: logentries/README.md [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+Lint the chart
+
+```
+$ helm lint
+```
+
+To test the generated k8s resources from the helm chart
+
+```
+$ helm upgrade --install --dry-run --debug my-new-chart  .
```
Comment (andrii-kovalenko-ct) [inline logentries/README.md]: shouldn't it be `helm upgrade --install ....` ?

File: logentries/values.yaml [hunk 4]
```diff
@@ -0,0 +1,11 @@
+image:
+  name: logentries
```
Comment (andrii-kovalenko-ct) [inline logentries/values.yaml]: @svenmueller @heshamMassoud 
this property _image.name_ looks confusing: actually it is not an image name, but rather application or container name, see https://github.com/commercetools/k8s-charts/pull/2/files#diff-c65d7160081d712e179b5ec0a5d1c1b0R13

Also in the doc: https://github.com/commercetools/k8s-charts/pull/2/files#diff-b5ebfd966867ca8860da5fe8e7f66888R40

Comment (svenmueller) [reply logentries/values.yaml]: @andrii-kovalenko-ct it's not required at all (see my comments in the PR for the crobjob chart)

File: logentries/README.md (hunk 2 above)
Comment (LEQADA) [reply logentries/README.md]: https://github.com/commercetools/k8s-charts/pull/2/commits/78a1004482ec8d4281e2f2b69ff9e97199ed0538

File: logentries/README.md (hunk 3 above)
Comment (LEQADA) [reply logentries/README.md]: @andrii-kovalenko-ct what do you mean? Or is it already solved?

File: logentries/templates/daemon-set.yaml (hunk 1 above)
Comment (LEQADA) [reply logentries/templates/daemon-set.yaml]: https://github.com/commercetools/k8s-charts/pull/2/commits/c25f1027d8bb139f493c19709d1e86454b243463

File: logentries/README.md (hunk 3 above)
Comment (andrii-kovalenko-ct) [reply logentries/README.md]: guess i commented on old version

**Comment by svenmueller** [comment]
please move the chart folder to subfolder `/charts`

File: charts/logentries/README.md [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+
+To test the generated k8s resources from the helm chart
+
+```
+$ helm upgrade --install --dry-run --debug my-new-chart  .
+```
+
+### Installation
+
+To install the chart with the release name `my-logentries` using a dedicated namespace(recommended):
```
Comment (svenmueller) [inline charts/logentries/README.md]: nitpick: whitespace missing after "namspace"

**Review by svenmueller — APPROVED** [review]
Looks nice!

**Review by LEQADA — APPROVED** [review]

File: charts/logentries/README.md (hunk 5 above)
Comment (heshamMassoud) [reply charts/logentries/README.md]: eec911d

**Pull request merged**